### DISCUSSION TITLE
MSMQ Quickstart

### DISCUSSION THREAD

**Discussion by jdezego** [newrelic/newrelic-quickstarts#1120] [Q&A]
Can someone clarify this QuickStart for me? As far as I can tell it does nothing to show metrics from MSMQ. Even the doc page below, although it says MSMQ the charts only show the usual transaction information from a dotnet application.

https://newrelic.com/instant-observability/msmq/8a2e0510-8a26-4a61-bd41-512dff1e2f01

If I'm monitoring MSMQ I want to see number of queues, number of messages etc. This is exactly what the MSMQ On-host integration does. So what is the purpose of this MSMQ QuickStart?

**Comment by seemantk** [comment]
Hi @jdezego, thank you for your patience, as I'm new here.

You're correct, the dashboard contained there is a generic dashboard.  Our team will prioritize this one for development, and I'll happily update you when it is ready.

**Reply by jdezego** [reply]
Thank you.

**Reply by Rik742** [reply]
Hi @seemantk, this is still the same situation after 2 years, can you update us?
